src/content/docs/deployment/configuration.mdx

```diff
@@ -33,6 +33,25 @@ If enabled, GPU will be required for rendering. If only CPU based adapters will
     - `true` or `1`, to enable
     - `false` or `0`, to disable
 
+### `SMELTER_GPU_DEVICE_ID`
+ID of a GPU device to use. The IDs can be found in the startup logs, when adapters are logged, described as `device` field:
+```
+ 2025-09-30T12:14:48.113571Z  INFO smelter_core::graphics_context::vulkan_context Available adapters:
+ - AdapterInfo { name: "AMD Radeon RX 7800 XT (RADV NAVI32)", device_type: DISCRETE_GPU, api_version: "1.4.318" , driver: "radv", driver_info: "Mesa 25.2.4-arch1.1",  vendor: 4098, device: 29822, supports_decoding: true, supports_encoding: true }
+ - AdapterInfo { name: "AMD Radeon Graphics (RADV RAPHAEL_MENDOCINO)", device_type: INTEGRATED_GPU, api_version: "1.4.318" , driver: "radv", driver_info: "Mesa 25.2.4-arch1.1", vendor: 4098, device: 5710, supports_decoding: true, supports_encoding: true }
+```
+
+Can be used in combination with [`SMELTER_GPU_DEVICE_DRIVER`](#smelter_gpu_device_driver) if the same device is supported by multiple drivers.
+
+### `SMELTER_GPU_DEVICE_DRIVER`
+Filters out GPU devices which don't support the provided driver.
+The devices and their drivers can be found in the startup logs, when adapters are logged, described as `driver` and `driver_info` fields.
```

Comment:
```suggestion
Selects GPU device by the driver name or version.

The devices and their drivers can be found in the startup logs, when adapters are logged, described as `driver` and `driver_info` fields.
```

btw, please don't write markdown like this. Either no newlines, newlines based on column width, or more than one newline. Single newline does not change anything for the output, but it gives false impression of the text layout when looking at the raw markdown